helix-term/src/ui/picker.rs

```diff
@@ -1143,6 +1173,12 @@ impl<I: 'static + Send + Sync, D: 'static + Send + Sync> Component for Picker<I,
             ctrl!('t') => {
                 self.toggle_preview();
             }
+            ctrl!('j') => {
+                self.scroll.position += 1;
```

Comment:
This needs to be capped at `doc.len_lines()`. Scrolling past the end of the document causes a panic.